pkg/macvlan/macvlan.go

```diff
@@ -270,6 +271,79 @@ func macvlanCmdDel(args *skel.CmdArgs) error {
 	return err
 }
 
+// validatePortRange validates the destination port value provided in the NAD.
```

Comment:
As this (and the following) are "util" functions, it's good practice to have them after the function that consumes them (in this case macVlanCmdAdd), so the reader finds the "important" stuff first

Reply:
Will move this in the next iteration when adding unit tests.